api-reference/_hidden/PdfExportGanttFont/PdfExportGanttFont.md

```diff
@@ -0,0 +1,23 @@
+---
+id: PdfExportGanttFont
+---
+---
+##### shortDescription
+Configures a custom font used for the Gantt data export.
+
+---
+
+The exporter supports standard PDF fonts. Specify the [font](/Documentation/ApiReference/Common/Object_Structures/PdfExportGanttProps/#font) property in the [exportGantt](/Documentation/ApiReference/Common/Utils/pdfExporter/#exportGanttoptions) method to use a custom font in the exported Gantt data.
+
+    <!-- tab: JavaScript -->
+    function exportGantt() {
+        DevExpress.pdfExporter.exportGantt({
```

Comment:
This isn't framework-specific. It's better to give the example similarly to [this article](https://js.devexpress.com/Documentation/ApiReference/Common/Utils/utils/#getTimeZonesdate).